src/helpers/performance-report-analyser.helper.js

```diff
@@ -0,0 +1,36 @@
+import fs from 'fs';
+import chalk from 'chalk';
+
+class PerformanceReportAnalyser {
+  getReport(fileName) {
+    return JSON.parse(fs.readFileSync(`reports/performance/${fileName}`, 'utf8'));
+  }
+
+  parse(fileName) {
```

Comment:
The whole parsing thing can be moved to a Parsers class. Create a JSONPerformanceReportParser with a single method "parse" that will have this method inside and keep "checkTiming" method here only. 

You can then inject Parser  as a dependency for Analyser -> will be much easier for you to test analyse method.